{"name": "test_login", "status": "broken", "statusDetails": {"message": "TypeError: 'NoneType' object is not callable", "trace": "self = <tests.test_login.TestLogin object at 0x76ede0a13ed0>\ninitialize_driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"7b4cf0cc16c60ffbfe26c53dc99dae30\")>\nlogger_creation = None\n\n    def test_login(self, initialize_driver,logger_creation):\n      home_page = HomePage(initialize_driver)\n>     logger_creation()\nE     TypeError: 'NoneType' object is not callable\n\ntests/test_login.py:18: TypeError"}, "start": 1732029450540, "stop": 1732029450542, "uuid": "495ff36e-0787-49aa-a334-8fb3b732c5be", "historyId": "2116f0ad00ab1de0effc54f114dd9496", "testCaseId": "2116f0ad00ab1de0effc54f114dd9496", "fullName": "tests.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "aelita-hp-pavilion"}, {"name": "thread", "value": "48914-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}